src/pages/404/404.tsx

```diff
@@ -0,0 +1,30 @@
+import { Link, route } from 'preact-router';
+import { useEffect } from 'preact/hooks';
+import styles from "./404.module.sass";
+
+//TODO: add an arrow in the primary button
+
+interface Props {
+  closeNav: Function;
+}
+const NotFound = (props: Props) => {
+  useEffect(() => {
+    props.closeNav();
+  }, []);
+  const returnHome = () => {
+    route('/');
```

Comment:
Function can be removed (it's only one line of code).
Also, for changing routes you need to add `true` as the second parameter of this function, so it'll rewrite the URL.